terraform/utils.go

```diff
@@ -87,6 +87,17 @@ func tfConfigList(ctx context.Context, d *plugin.QueryData, _ *plugin.HydrateDat
 
 	// Sanitize the matches to likely Terraform files
 	for _, i := range matches {
+		// Check file or directory
+		fileInfo, err := os.Stat(i)
+		if err != nil {
```

Comment:
What type of errors can we expect here?

Reply:
@cbruno10 This block will mostly use to handle `no such file or directory` error.